Ports/stpuzzles/package.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SimonTathamsPuzzles
+useconfigure=true
+version=1.0
+workdir=stpuzzles-serenity
+configopts="-DCMAKE_TOOLCHAIN_FILE=$SERENITY_SOURCE_DIR/Toolchain/CMake/CMakeToolchain.txt"
+files="https://github.com/Cleverking2003/stpuzzles/archive/refs/tags/serenity-${version}.zip stpuzzles.zip b9fcacbbc1a542f85fd3e62095aab52f88b6ec26533479103b55135ef822de08"
```

Comment:
Unless there's a very good reason for it, please don't use a personal fork for ports sources. Use either a upstream repository/tarball/whatever or a repository under the SerenityOS GitHub organization.

Reply:
@linusg The changes I made to the original are too big to be made using patches and I can't create repositories in SerenityOS organization. What should I do?

Reply:
Would you be happy to [transfer](https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/transferring-a-repository) the repo? If not, I suppose we could fork it...

Reply:
> Would you be happy to [transfer](https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/transferring-a-repository) the repo? If not, I suppose we could fork it...

That's alright, I'll transfer it.

Reply:
@linusg It seems that I can't transfer the repo because I'm not a member of the organization.